polaris-server.yml

```diff
@@ -167,3 +167,5 @@ logging:
       archivedLogFilenamePattern: ./logs/polaris-%d.log.gz
       # The maximum number of log files to archive.
       archivedFileCount: 14
+
+maxDocumentBytes: 0 # Don't set a limit
```

Comment:
ditto

Reply:
Maybe use -1 to indicate no limit since 0 can be used to disallow anything?

Reply:
We may also explain what is the default value.

Reply:
Please see my new comment below on why 0 may make more sense in this specific instance.